Frontend/shared/src/functions/deleteShoppingList/deleteShoppingList.ts

```diff
@@ -1,4 +1,4 @@
-import { BASE_URL } from '@constants/config';
+import { BASE_URL } from '@constants';
 /*
```

Comment:
This comment can be removed because we can just look at the definition of a GroceryItem.